### PR TITLE
profile page designed till navigation bar along with css

### PR DESCRIPTION
**Pull request by abharana24**

### CODE REVIEW

File: Abha_Rana/Tic-Tac-Toe/tt.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
+* {
+    box-sizing: border-box;
```
Comment (nikhilrstg18) [inline Abha_Rana/Tic-Tac-Toe/tt.css]: Proper indentation required for every ruleset

File: Abha_Rana/Tic-Tac-Toe/tt.css [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    padding: 0;
+    margin: 0;
+  }
+  .container {
+    display: flex;
+    background: #fff;
+    align-content: center;
+    justify-content: center;
+    flex-direction: column;
+    min-width: 700px;
```
Comment (nikhilrstg18) [inline Abha_Rana/Tic-Tac-Toe/tt.css]: You are using px as unit,  :)
Is the container responsive?

File: Abha_Rana/Tic-Tac-Toe/tt.css [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+  }
+  .container {
+    display: flex;
+    background: #fff;
+    align-content: center;
+    justify-content: center;
+    flex-direction: column;
```
Comment (nikhilrstg18) [inline Abha_Rana/Tic-Tac-Toe/tt.css]: it is recommended to use shorthand for writing CSS declaration improve readability
for eg. use flex-flow instead as below

flex-flow: <flex-direction> <flex-wrap>;

File: Abha_Rana/Tic-Tac-Toe/tt.css [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    position: absolute;
+      top:0;
+      width:100%;
+  }  
+  .heading{
+      width:100%;
+      text-align: center; 
+     }
+  
+  table {
```
Comment (nikhilrstg18) [inline Abha_Rana/Tic-Tac-Toe/tt.css]: never apply styling to an element in the main CSS file
start implementing the power of specificity

**Review by nikhilrstg18 — CHANGES_REQUESTED** [review]
Missing SOLID DRY KISS in code
Missing comment for displaying purpose of each function in JS

File: Abha_Rana/my_profile/main.html [hunk 5]
```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset='utf-8'>
+    <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+    <title>Page Title</title>
+    <meta name='viewport' content='width=device-width, initial-scale=1'>
+    <link rel='stylesheet' type='text/css' media='screen' href='style.css'>
```
Comment (nikhilrstg18) [inline Abha_Rana/my_profile/main.html]: still missing folder structure

File: Abha_Rana/my_profile/main.html [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+              <li><a href="#where">WHERE</a></li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+        <!-- First Container -->
+        <div class="container-fluid bg-1 text-center" id="who">
+            <h1 class="margin">Who Am I?</h1>
+            <div class="row">
+                <div class="col-sm-4 zoom" style="background-color:palevioletred;">
```
Comment (nikhilrstg18) [inline Abha_Rana/my_profile/main.html]: get rid of style attribute from html files

File: Abha_Rana/my_profile/style.css [hunk 7]
```diff
@@ -0,0 +1,69 @@
+body {
```
Comment (nikhilrstg18) [inline Abha_Rana/my_profile/style.css]: missing consistency in indentation

File: Abha_Rana/my_profile/style.css [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+            font-family: 'Gloria Hallelujah', cursive;
+            
+          }
+.nav
+        {
+            font-family: 'Pacifico', cursive;
+        }
+.col-sm-4
+        {
+            border-style:dotted;
```
Comment (nikhilrstg18) [inline Abha_Rana/my_profile/style.css]: use shorthand for border

File: Abha_Rana/my_profile/style.css [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+      color: #fff;
+    }
+.container-fluid {
+            padding-top: 25px;
+            padding-bottom: 60px;
+        
+          }
+ .navbar{
+            position:sticky;
+            padding-top: 5px;
```
Comment (nikhilrstg18) [inline Abha_Rana/my_profile/style.css]: use shorthand for padding

File: Abha_Rana/Tic-Tac-Toe/assets/scripts/ttt.js [hunk 10]
```diff
@@ -0,0 +1,112 @@
...
+    }
+  }
+  return false;
+}
+
+/**
+ * Helper function to check if NodeList from selector has a particular text
+ */
+function contains(selector, text)
+  var elements = document.querySelectorAll(selector);
```
Comment (nikhilrstg18) [inline Abha_Rana/Tic-Tac-Toe/assets/scripts/ttt.js:82]: what is purpose of queryselector ?

File: Abha_Rana/Tic-Tac-Toe/assets/scripts/ttt.js [hunk 11]
```diff
@@ -0,0 +1,112 @@
...
+  return [].filter.call(elements, function(element) {
```
Comment (nikhilrstg18) [inline Abha_Rana/Tic-Tac-Toe/assets/scripts/ttt.js:83]: why can't we call `filter` directly on `elements`?

**Review by nikhilrstg18 — CHANGES_REQUESTED** [review]
minor code changes requested, please reply on each comment added

File: Abha_Rana/Tic-Tac-Toe/tt.css (hunk 3 above)
Comment (abharana24) [reply Abha_Rana/Tic-Tac-Toe/tt.css]: Issue fixed.  used flex-flow instead of flex-direction.

File: Abha_Rana/Tic-Tac-Toe/tt.css (hunk 2 above)
Comment (abharana24) [reply Abha_Rana/Tic-Tac-Toe/tt.css]: used percent instead of px for responsiveness.

File: Abha_Rana/Tic-Tac-Toe/tt.css (hunk 1 above)
Comment (abharana24) [reply Abha_Rana/Tic-Tac-Toe/tt.css]: indentation is done via prettier.

File: Abha_Rana/Tic-Tac-Toe/tt.css (hunk 4 above)
Comment (abharana24) [reply Abha_Rana/Tic-Tac-Toe/tt.css]: elements are created in the script and hence do have any id or class. hence I had to apply styling to only those elements created using javascript.

File: Abha_Rana/Tic-Tac-Toe/assets/scripts/ttt.js (hunk 10 above)
Comment (abharana24) [reply Abha_Rana/Tic-Tac-Toe/assets/scripts/ttt.js:82]: queryselectorall() gets all the html elements that match the specified css selector which in this case are all the elements having id="tictactoe". it is further used with .length property to get the number of elements having that id and extract information present in them by traversing them via loops.

File: Abha_Rana/my_profile/style.css (hunk 7 above)
Comment (abharana24) [reply Abha_Rana/my_profile/style.css]: resolved

File: Abha_Rana/my_profile/main.html (hunk 5 above)
Comment (abharana24) [reply Abha_Rana/my_profile/main.html]: folder structure updated.

File: Abha_Rana/my_profile/assets/css/style.css [hunk 12]
```diff
@@ -0,0 +1,69 @@
+body {
```
Comment (nikhilrstg18) [inline Abha_Rana/my_profile/assets/css/style.css]: poor indentation  --> poor readability
fix to the proper indentation in all files

File: Abha_Rana/my_profile/style.css (hunk 7 above)
Comment (nikhilrstg18) [reply Abha_Rana/my_profile/style.css]: This is still not fixed

File: Abha_Rana/my_profile/main.html (hunk 6 above)
Comment (nikhilrstg18) [reply Abha_Rana/my_profile/main.html]: Still not fixed

**Review by nikhilrstg18 — CHANGES_REQUESTED** [review]
still few PR comments are not addressed with proper replies, check and fix more added

File: Abha_Rana/my_profile/main.html (hunk 6 above)
Comment (abharana24) [reply Abha_Rana/my_profile/main.html]: got rid of inline styling from html files

File: Abha_Rana/my_profile/style.css (hunk 9 above)
Comment (abharana24) [reply Abha_Rana/my_profile/style.css]: used padding instead of padding-top. issue fixed.

File: Abha_Rana/my_profile/style.css (hunk 8 above)
Comment (abharana24) [reply Abha_Rana/my_profile/style.css]: used shorthand for border.  issue resolved.

File: Abha_Rana/my_profile/style.css (hunk 7 above)
Comment (abharana24) [reply Abha_Rana/my_profile/style.css]: fixed.

File: Abha_Rana/Tic-Tac-Toe/assets/scripts/ttt.js (hunk 11 above)
Comment (abharana24) [reply Abha_Rana/Tic-Tac-Toe/assets/scripts/ttt.js:83]: filter.call -Uses a blank array in order to call the "filter" method
Tells the filter to use list as the array to filter, and uses function(elements) to do the filtering, where  "elements" is the item in the list being tested
If the item has a style attribute that is empty, i.e. has no style applied, it returns true.

File: Abha_Rana/my_profile/assets/css/style.css (hunk 12 above)
Comment (abharana24) [reply Abha_Rana/my_profile/assets/css/style.css]: fixed

**Review by nikhilrstg18 — APPROVED** [review]
looks good to me

**Review by amandeeprikhi — APPROVED** [review]
LGTM

**Comment by vishivish18** [comment]
@abharana24  please fix the conflict and ask for as mergr

**Pull request merged**